examples/android_app/app/src/main/java/com/example/llamastackandroiddemo/ETImage.java

```diff
@@ -0,0 +1,126 @@
+/*
```

Comment:
Why is this file in here?

Reply:
I should remove this for now to avoid confusion. But will add back soon

Reply:
Removed